README.md

```diff
@@ -44,7 +44,7 @@ The only algorithms in `liboqs` that implement NIST standards are the [`ML-KEM`]
 
 Falcon and SPHINCS+ have also been [selected for standardization](https://csrc.nist.gov/Projects/post-quantum-cryptography/selected-algorithms-2022), but the `liboqs` implementations of these algorithms are currently tracking Round 3 submissions and not NIST standards drafts.
 
-All names other than `ML-KEM` and `ML-DSA` are subject to change. `liboqs` makes available a [selection mechanism for algorithms on the NIST standards track, continued NIST competition, or purely experimental nature by way of the configuration variable OQS_ALGS_ENABLED](CONFIGURE.md#oqs_algs_enabled). By default `liboqs` is built supporting all, incl. experimental, PQ algorithms listed below.
+All names other than `ML-KEM` and `ML-DSA` are subject to change. `liboqs` makes available a [selection mechanism for algorithms on the NIST standards track, continued NIST competition, or purely experimental nature by way of the configuration variable OQS_ALGS_ENABLED](CONFIGURE.md#oQS_ALGS_ENABLED). By default `liboqs` is built supporting all, incl. experimental, PQ algorithms listed below.
```

Comment:
No, that's exactly the problem, as far as I can tell: Doxygen expects its auto-generated anchor tags to be capitalized in this exact way, with the first letter always lowercase, and then the rest of the letters as given.

Reply:
Just checking: do we expect these links to work in the generated HTML documentation? For me they seem to be broken in the HTML docs, but that's because they refer to the non-HTML file CONFIGURE.md.

Reply:
I'm not sure how to work around this -- we want the links in Github's rendering of the Markdown to point to the .md file, and we want the links in Doxygen's rendering to point to the .html file.  The only way I can think of to fix this is a preprocessing script in the `gen_docs` target that edits the file before Doxygen is run. I'm not sure it's worth the effort to fix at the moment.

Reply:
I'm inclined to agree and would endorse merging this as-is as per @mouse07410's [comment below](https://github.com/open-quantum-safe/liboqs/pull/1927#pullrequestreview-2319955683).

Reply:
Okay, resolving conversation as won't-fix.